packages/cardano-services/src/ProgramsCommon/CommonOptionDescriptions.ts

```diff
@@ -1,5 +1,5 @@
 export enum CommonOptionDescriptions {
-  CacheTtl = 'Cache TTL in minutes between 1 and 2880',
+  DbCacheTtl = 'Cache TTL in minutes between 1 and 2880, an option for database related operations',
```

Comment:
Would be nice to list what exactly is being cached

Reply:
I think the list would become longer for this static description, as currently, we cache 4 network info related values, 1 epoch-related, and soon more stake-pool related cached values.

Reply:
Probably could be created a cache description generation util which gets a config list of cached keys and returns a composed description, not sure how it will look with `cli --help` though :D

Reply:
Yeah, nevermind if it's not practical